docker/build-docker.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+set -o errexit
+
+realpath() {
+  echo "$(cd "$(dirname "$1")"; pwd)/$(basename "$1")"
+}
+
+DOCKER_DIR=$(dirname $(realpath $0))
+PROJECT_DIR=$(dirname $DOCKER_DIR )
+SCRATCH_DIR="$HOME/.heron-docker"
+
+cleanup() {
+  if [ -d $SCRATCH_DIR ]; then
+    echo "Cleaning up scratch dir"
+    rm -rf $SCRATCH_DIR
+  fi
+}
+
+trap cleanup EXIT
+
+
```

Comment:
Yes, I was trying to make this consistent-ish with the other scripts there. If there is a more specific script style I can adjust it.